com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1582,9 +1629,13 @@ public override void Initialize(NetworkManager networkManager = null)
 
             m_NetworkManager = networkManager;
 
-            if (m_NetworkManager && m_NetworkManager.PortOverride.Overidden)
+            if (!m_HasForcedConnectionData && ParseCommandLineOptions(out var port))
             {
-                ConnectionData.Port = m_NetworkManager.PortOverride.Value;
+                if (m_NetworkManager?.LogLevel <= LogLevel.Developer)
+                {
+                    Debug.Log($"Already has command line option set. Using connection data set to {ConnectionData.Address}:{port}");
+                }
+                ConnectionData.Port = (ushort)port;
```

Comment:
```suggestion
                ConnectionData.Port = port;
```

This will fix your standards check error. ParseCommandLineOptions is already defining the type written to in the 1st out parameter.

If you opt to make a more generic one, then you might adjust ParseCommandLineOptions to be something like ParseCommandLineOptions<T>.